kernels/absolute.cl

```diff
@@ -0,0 +1,18 @@
+__constant sampler_t sampler = CLK_NORMALIZED_COORDS_FALSE | CLK_ADDRESS_CLAMP_TO_EDGE | CLK_FILTER_NEAREST;
+
+__kernel void absolute(
+    IMAGE_src_TYPE  src,
+    IMAGE_dst_TYPE  dst
+)
+{
+  const int x = get_global_id(0);
+  const int y = get_global_id(1);
+  const int z = get_global_id(2);
+
+  float value = READ_IMAGE(src, sampler, POS_src_INSTANCE(x,y,z,0)).x;
+  if ( value < 0 ) {
+    value = -1 * value;
+  }
+
+  WRITE_IMAGE (dst, POS_dst_INSTANCE(x,y,z,0), CONVERT_dst_PIXEL_TYPE(value));
```

Comment:
Is there an impact (speed, conversion, etc.) on using `float` to store the value here or should we avoid hard-coded type and rely on the `IMAGE_dst_PIXEL_TYPE` and  `CONVERT_dst_PIXEL_TYPE` ? 

```suggestion
  IMAGE_dst_PIXEL_TYPE value = CONVERT_dst_PIXEL_TYPE(READ_IMAGE(src, sampler, POS_src_INSTANCE(x,y,z,0)).x);
  if ( value < 0 ) {
    value = -1 * value;
  }

  WRITE_IMAGE (dst, POS_dst_INSTANCE(x,y,z,0), value);
```